### ISSUE TITLE
[Question] Populate Definitions - What is "params" used for?

### ISSUE THREAD

**Issue by kshitizshankar** [marshallswain/feathers-graph-populate#12]
First of all - Thanks a lot for pointing me to this earlier! I am exploring the library now and the overall ease of defining the relationships and named queries makes my whole flow so much cleaner. I've been using a mix of batch loaders and populates awkwardly to achieve what this library does out of the box in a much much better way!

Question - What exactly is the "params" in the populate definitions used for? Couldn't find any reference/documentation of that.
```
orgMemberships: {
    service: 'org-users',
    nameAs: 'orgMemberships',
    keyHere: '_id',
    keyThere: 'userId',
    asArray: true,
    params: {}
  }
```

Thanks!

**Comment by marshallswain** [comment]
That entire object is handled by `feathers-shallow-populate`.  It's undocumented there as well, but it basically allows you to specify the params object sent along in the feathers request.  What you specify gets merged into the params the `feathers-shallow-populate` sets up, internally.

**Comment by kshitizshankar** [comment]
Think that PR from @fratzinger is the exact thing I was arriving at. I needed to pass a function/dynamic query param based on the hook context to hook my authorization flow within it.

**Comment by marshallswain** [comment]
I've just published `feathers-graph-populate@1.1.0` with @fratzinger's shallow-populate PR merged in. The docs could use an update, now.
